pkgs/development/python-modules/face-recognition/default.nix

```diff
@@ -36,6 +36,9 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  # Disable tests
+  doCheck = false;
```

Comment:
Last nit! Can you make this `doCheck = !cudaSupport` please

Reply:
Oh, I thought this was in `dlib`. Well, `dlib.cudaSupport` would've made sense still, but maybe out of scope

Reply:
I can still add it, I probably should put a reference to the issue as well, might help someone else who comes across it.